build/changelog/ChangelogBuilder.php

```diff
@@ -0,0 +1,149 @@
+<?php
+namespace Aws\Build\Changelog;
+
+/**
+ * @internal
+ */
+
+class ChangelogBuilder 
+{
+  
+  public $newServiceFlag = False;
```

Comment:
`false` should be lower-cased. You might want to run a style fixer (e.g., http://cs.sensiolabs.org/) on this file to get everything PSR-2 compliant.